__tests__/app.test.js

```diff
@@ -89,4 +89,39 @@ describe("/api/articles/:article_id", () => {
             expect(response.body.msg).toBe("Bad request")
         })
     })
-})
\ No newline at end of file
+})
+describe("GET /api/articles", () => {
+    test("should respond with an array of all article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+            expect(articles).toHaveLength(13)
+            articles.forEach((article) => {
+                 expect(Object.keys(article)).toIncludeSameMembers([
+                    "author",
+                    "title",
+                    "article_id",
+                    "topic",
+                    "created_at",
+                    "votes",
+                    "article_img_url",
+                    "comment_count",
+                ])
+            })
```

Comment:
I mean, this test would work but in terms of readability it's not great imo. We need to read using Object.keys to get an array and then the matcher toIncludeSameMembers then an array or the object keys.

We already have all the properties of the article object so can't we just check to see if that property exists? It would make it much more readable.

Reply:
Okay, I will change the test. Thank you